workers/unity/Packages/io.improbable.gdk.testutils/Editor/SpatialdManager.cs

```diff
@@ -191,7 +191,17 @@ public async Task<List<LocalDeployment>> GetRunningDeployments()
                 .RunAsync()
                 .ConfigureAwait(false);
 
+            if (result.ExitCode != 0)
+            {
+                throw new Exception($"Failed to list deployments with error:\n {string.Join("\n", result.Stderr)}");
+            }
+
             var json = Json.Deserialize(string.Join("", result.Stdout));
+            if (json == null)
+            {
+                throw new Exception($"Failed to list deployments with error:\nEmpty output\n {string.Join("\n", result.Stderr)}");
```

Comment:
```suggestion
                throw new Exception($"Failed to list deployments because there was no output. Error: \n {string.Join("\n", result.Stderr)}");
```

Reply:
Just to make it a little more distinct 😉

Reply:
done